src/main/java/org/broadinstitute/dsm/model/PDF/TissueCoverPDF.java

```diff
@@ -0,0 +1,59 @@
+package org.broadinstitute.dsm.model.PDF;
+
+import lombok.NonNull;
+import org.apache.commons.lang3.StringUtils;
+import org.broadinstitute.dsm.db.DDPInstance;
+import org.broadinstitute.dsm.db.OncHistoryDetail;
+import org.broadinstitute.dsm.db.dto.user.UserDto;
+import org.broadinstitute.dsm.files.RequestPDFProcessor;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.Map;
+
+public class TissueCoverPDF{
+    Logger logger = LoggerFactory.getLogger(TissueCoverPDF.class);
+    private DownloadPDF originalDownloadPDF;
+
+    public TissueCoverPDF(@NonNull DownloadPDF downloadPDF){
+        this.originalDownloadPDF = downloadPDF;
+    }
+
+    public Map<String, Object> getValuesForTissueCover(DDPInstance ddpInstance, UserDto user) {
+
+        logger.info("Generating request pdf for onc history ids {}", StringUtils.join(originalDownloadPDF.oncHistoryIDs, ","));
+        Map<String, Object> valueMap = new HashMap<>();
+        String today = new SimpleDateFormat("MM/dd/yyyy").format(new Date());
```

Comment:
again, can we move the format string, maybe use the same in all pdf classes or from system utils or utility class